PWGHF/TableProducer/candidateSelectorD0.cxx

```diff
@@ -85,6 +86,11 @@ struct HfCandidateSelectorD0 {
 
   void init(InitContext& initContext)
   {
+    std::array<bool, 2> doprocess{doprocessWithDCAFitterN, doprocessWithKFParticle};
```

Comment:
since you kindly do in the `taskD0`, what about adding also here a check that throws a `FATAL` in case no `process` functions are activated?

Reply:
Thanks for the comments.
As mentioned by Vit, the case where no `process` functions are activated should be included by the `!=1`

Reply:
you are right, sorry, I got confused by the message